lib/adapters/amqp/index.ts

```diff
@@ -10,7 +10,7 @@ import {AMQPPublisherAdapter} from "./publisher";
 import {AMQPSubscriberAdapter} from "./subscriber";
 
 const AMQP = require("amqp-coffee");
-const _ = require("lodash");
+const clone = require("lodash.clone");
```

Comment:
@Kumjami in this case the config _is_ an object. So we should not assume it might be an array.
Given that _fact_ , [clone](https://github.com/lodash/lodash/blob/master/clone.js) will behave the same way as `Object.assign`. => `The own enumerable properties of `arguments` objects are cloned`